mobile/src/lib/cozy-helper.js

```diff
@@ -73,3 +75,17 @@ export function refreshFolder (dispatch, getState) {
     }
   }
 }
+
+export const onError = (dispatch, getState, callback) => (err) => {
+  if (err.message === clientRevokedMsg) {
+    console.warn(`Your device is no more connected to your server: ${getState().mobile.settings.serverUrl}`)
+    dispatch(revokeClient())
+  } else if (err.message === 'ETIMEDOUT' || err.message === 'getCheckpoint rejected with ') {
+    console.log('timeout')
+  } else {
+    console.warn(err)
+  }
+  if (callback) {
```

Comment:
If you accept all the others comments, then you should remove `callback` from this function. Cf. https://github.com/cozy/cozy-files-v3/pull/179/files#r106179418